src/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -45,18 +45,19 @@ export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
     <Menu onClick={handleClick} width={width} m={0} py={0}>
       <Flex justifyContent="center">
         <SimpleLinksContainer
-          py={4}
+          mt={0.5}
+          py={3}
           mr={[3, 3, 3, "50px"]}
           viewAllTopMargin={viewAllTopMargin[contextModule]}
         >
-          <Box mr={[2, 2, 3, 3]} width={[110, 110, 110, 135, 150]}>
+          <Box mr={[1, 1, 2, 2]} width={[110, 110, 110, 135, 170]}>
```

Comment:
I bumped up the with for larger breakpoints here so that text would fit on two lines per the specs.